### DISCUSSION TITLE
BaramMesh Invalid coordinates ???

### DISCUSSION THREAD

**Discussion by DirkDali** [nextfoam/baram#63] [General]
![Bildschirmfoto zu 2024-03-24 10-57-15](https://github.com/nextfoam/baram/assets/163728849/6bd5a2df-b6dc-40d8-9905-0fcc38de14f9)
In my opinion, all coordinates are positive and therefore correct.
Furthermore, when loading the *.stl models, the split surface does not work.
Can anyone tell me if I'm doing something wrong here or if this is a program error?
![split](https://github.com/nextfoam/baram/assets/163728849/23be90ba-4b27-462e-ab9c-8e61065f5857)
Thanks
Dirk

**Comment by jiban** [comment]
Hi, Dirk.
Could you upload the STL files for me to check what went wrong?

**Comment by DirkDali** [comment]
Hi Jiban,
 i try the upload but this chat don´t support upload file type *.stl 
Greetings Dirk

**Comment by jiban** [comment]
Then you can send it to marketing@nextfoam.co.kr.

**Reply by DirkDali** [reply]
OK, you habe the files!
Greetings Dirk

**Comment by jiban** [comment]
Hi, Dirk.
I got the files, but I could not reproduce your problem.
It was just imported well and split with feature angle of 60 like following.

![image](https://github.com/nextfoam/baram/assets/1674177/fa148a5a-4d73-4199-9f26-dc814b9d8961)

However, I found that the files you sent me were different from the files you tried.
They have different names - "2" as suffix in your case - and different dimensions.
Please check if you sent me the right files.

-- Jake

**Comment by DirkDali** [comment]
Hi Jiban,
You're right, I sent you other files but I had the same problem with them. I think the problem is with my Linux installation.
Thanks for your support.
I'll install everything again.
Best regards
Dirk
